usage/node-placement.md

```diff
@@ -143,3 +143,16 @@ An example for `tolerations` may look like this:
         operator: "Equal"
         value: "value"
         effect: "NoSchedule"
+
+Node balancing with Descheduler
+----------------------
+In some cases we might need to rebalance the cluster on current scheduling policy 
+and load conditions. [Descheduler](https://github.com/kubernetes-sigs/descheduler) 
+can find pods, which violates e.g. scheduling decisions and evict them based on descheduler 
+policies. Kubevirt VMs are handled as pods with local storage, so by default, 
+descheduler will not evict them. But it can be easily overridden by adding special annotation to vm:
+
+      descheduler.alpha.kubernetes.io/evict: true
+
+This annotation will case, that descheduler will be able to evict VM's pod which can be then 
+scheduled by scheduler on different nodes. Users should know how and if the pod will be recreated.
```

Comment:
Question: how will users know how/if the pod will be recreated? Maybe add a reference here?